src/applications/disability-benefits/all-claims/utils/submit.js

```diff
@@ -362,7 +364,46 @@ export const addForm4142 = formData => {
   return clonedData;
 };
 
+export const delete0781FormData = formData => {
+  // Workaround to avoid eslint rule around mutating params/args
+  const data = formData;
+  delete data.events;
+  delete data.eventTypes;
+  delete data.workBehaviors;
+  delete data.otherBehaviors;
+  delete data.healthBehaviors;
+  delete data.workBehaviorChanges;
+  delete data.socialBehaviorChanges;
+  delete data.behaviorsDetails;
+  delete data.supportingEvidenceRecords;
+  delete data.supportingEvidenceReports;
+  delete data.supportingEvidenceUnlisted;
+  delete data.supportingEvidenceWitness;
+  delete data.supportingEvidenceOther;
+  delete data.supportingEvidenceNoneCheckbox;
+  delete data.optionIndicator;
+  delete data.additionalRemarks781;
+  delete data.treatmentNoneCheckbox;
+  delete data.additionalInformation;
+  data.vaTreatmentFacilities.forEach((facility, index, object) => {
+    if (facility.treatmentLocation0781Related) {
+      object.splice(index, 1);
```

Comment:
I dont think we want to DELETE the treatment center, just this value/key-value pair 🤔 

Like they could have a treatment center that they got 0781 related stuff and regular other stuff not related to 0781, so if this is deleting the location, I dont think it should

Reply:
Seconding Kyle's comment. If I understand this block correctly, all treatment facilities where `treatmentLocation0781Related` is **true** would be removed. However, that is not desired as those facility data populate other forms.

Reply:
I think you could, as Kyle suggested, simply remove any `treatmentLocation0781Related` key from each `facility`. The same should be performed for each `providerFacility`, which pertains to treatment received at non-VA facilities. 

You would also want to delete `treatmentReceivedVaProvider` and `treatmentReceivedNonVaProvider` data.